docs/concepts/workloads/vm.md

```diff
@@ -84,15 +84,98 @@ It is possible to update parts of an existing `VirtualMachine` resource. Some fi
+
+#### How are a VM's Network Interfaces Configured?
+VM operator uses the network devices specified in the VM Class to configure the network interfaces specified in the VM spec. If it cannot find sufficient number of devices in the VM class, a set of defaults are used to configure the network interfaces.
+
+As an example, consider the following VM Class that specifies two network devices, each with a different Ethernet card type.
+
+```yaml
+apiVersion: vmoperator.vmware.com/v1alpha1
+kind: VirtualMachineClass
+metadata:
+  name: class-with-two-nics
+spec:
+  configSpec:
+    _typeName: VirtualMachineConfigSpec
+    deviceChange:
+    - _typeName: VirtualDeviceConfigSpec
+      device:
+        _typeName: VirtualE1000
+        key: -100
+      operation: add
+    - _typeName: VirtualDeviceConfigSpec
+      device:
+        _typeName: VirtualVmxnet3
+        key: -200
+      operation: add
+  hardware:
+    cpus: 2
+    memory: 4Gi
+```
+
+If the following VM is created using the class defined above:
+```yaml
+apiVersion: vmoperator.vmware.com/v1alpha1
+kind: VirtualMachine
+metadata:
+  name: my-vm
+  namespace: my-namespace
+spec:
+  className: class-with-two-nics
+  imageName: ubuntu-kinetic
+  networkInterfaces:
+  - networkType: nsx-t
+    ethernetCardType: eth1000
+  - networkType: nsx-t
+    etherNetCardType: vmxNet3
+  - networkType: nsx-t
+    ethernetCardType: vmxnet2
+  - networkType: nsx-t
+  powerState: poweredOn
+  storageClass: wcpglobal-storage-profile
+```
+The first two network interfaces of the VM are configured using the VM class. So, even though they specify a card type, they inherit the `VirtualE1000` and `VirtualVmxnet3` types respectively. The third interface is configured using the card type it specifies - `VirtualVmxnet2`. The fourth interface does not specify any type, so the default Ethernet card type of `VirtualVmxnet3` is used.
+
 
 ### Storage
+A VM deployed using VM operator inherits the storage defined in the `VirtualMachineImage`.  However, developers can also provision and manage additional storage dynamically by leveraging `PersistentVolumes`. To do this, a user would create a `[PersistentVolumeClaim](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims)` resource by picking a `StorageClass` associated with their namespace, along with other properties such as the disk size, mode etc. VM operator then dynamically provisions a first class disks which is exposed to the guest as a block volume. Users can start using the disk after formatting and mounting it at a mountpoint. VM operator also supports resizing these volumes to increase their size.
```

Comment:
Remove the `` from [PersistentVolumeClaim](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims). It's ignoring the hyperlink otherwise.

Reply:
Done.  Thanks for spotting this.